src/extensions/cli/cli.ts

```diff
@@ -1,69 +1,32 @@
-'use strict'
+import { spawn } from 'node:child_process'
+import path from 'node:path'
+import { isNotNullish } from '../../utils/index.js'
 
-/**
- * The CLI helper used by the CLI extension.
- *
- * @module extensions/Cli/Cli
- */
-
-const { spawn } = require('child_process')
-const path = require('path')
+export type CliArgs = ConstructorParameters<typeof Cli>
 
-/**
- * Cli extension.
- *
- * @class
- */
 class Cli {
+    public cwd: string
+    public env: Record<string, string>
+    public killSignal: number | NodeJS.Signals | undefined
+    public killDelay: number
+    public exitCode: number | undefined
+    public stdout: string
+    public stderr: string
+
     constructor() {
-        /**
```

Comment:
Minor: I would let comment

Reply:
All the variables seems so clear. I don't think we really need explain what is env, cwd, stdout, stderr ...etc.

Well, I put them backs now.